configuration/reports/reportdescriptors/dataexports/hojaFrontal.yml

```diff
@@ -0,0 +1,22 @@
+key: "hojaFrontal"
+uuid: "abf3fa9a-6d64-11f0-ac4b-36a9bbfc3f4e"
+name: "Hoja Frontal"
+description: "Hoja Frontal"
+parameters:
+  - key: "patientId"
+    type: "java.lang.Integer"
+    label: "Patient ID"
```

Comment:
actually @mseaton, am I right that this accepts patient_id and not the user-facing identifier?  

Did you just add this for now until we know the best way?

Reply:
Well @dmdesimone , there are no requirements at all around it.  This was the easiest way to get it working.